src/styles/colour/index.md.njk

```diff
@@ -45,70 +48,40 @@ the GOV.UK colour palette when you update GOV.UK Frontend.
         <td class="app-colour-list-column app-colour-list-column--notes">
           {{colour.notes}}
         </td>
-
       </tr>
     {% endfor %}
-    
-
   {% endfor %}
   </tbody>
 </table>
 
-## Greyscale
-
-<table class="govuk-body app-colour-list" summary="Table of greyscale colours"> 
-
-  <tbody>
-  {% for colour in colours.greyscale %}
-
-    <tr class="app-colour-list-row">
-
-      <th class="app-colour-list-column app-colour-list-column--name">
-        <span class="app-swatch {% if colour.colour == "#ffffff" %}app-swatch-border{% endif %}" style="background-color:{{colour.colour}}"></span>
-        <code>{{colour.name}}</code>
-      </th>
-
-      <td class="app-colour-list-column app-colour-list-column--colour">
-        {{colour.colour}}
-      </td>
-      <td class="app-colour-list-column app-colour-list-column--notes">
-        {{colour.notes}}
-      </td>
-
-    </tr>
+## Colour palette
 
-  {% endfor %}
-  </tbody>
-</table>
+Use these colours for graphs and supporting material.
 
-## Extended colours
+To reference colours from the palette directly you should use the `govuk-colour`
+function.
 
-Use these colours for graphs and supporting material.
+Avoid using the palette colours directly when there is a contextual colour that
```

Comment:
```suggestion
Do not use the palette colours directly when there is a contextual colour that
```

Reply:
Will people link 'contextual colour' with Sass variable as we've described it above (I know the Sass variable is just the mechanism to GET the contextual colour, but it's not quite consistent with what we've said above).

Would "Do not reference colours from the palette directly if there is a Sass variable provided for your use case. For example..."

Reply:
I originally had 'Do not use' but changed it to 'Avoid' because it felt like it wasn't _that_ important for e.g. users of the Prototype Kit. Do we have any sort of convention on when to be 'hardline' ('Do not do this thing') and when to be a bit softer / more of a guide ('Avoid doing this thing')?